onnx/checker.cc

```diff
@@ -246,61 +412,142 @@ void check_attribute(
 
 void check_node(
     const NodeProto& node,
-    const CheckerContext& ctx,
+    const GraphProto& graph,
+    CheckerContext& ctx,
+    const LexicalScopeContext& lex_ctx) {
+  check_node(node, ctx, lex_ctx);
+
+  // Resolve domain for node
+  auto domain = node.domain();
```

Comment:
If domain is missing, we use default one.

Reply:
That's right. I've added an if() to guard this check.